drivers/adc/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ zephyr_library_sources_ifdef(CONFIG_ADC_ITE_IT8XXX2	adc_ite_it8xxx2.c)
 zephyr_library_sources_ifdef(CONFIG_ADC_SHELL		adc_shell.c)
 zephyr_library_sources_ifdef(CONFIG_ADC_MCUX_ADC12	adc_mcux_adc12.c)
 zephyr_library_sources_ifdef(CONFIG_ADC_MCUX_ADC16	adc_mcux_adc16.c)
-zephyr_library_sources_ifdef(CONFIG_ADC_MCUX_12B1MSPS_SAR 	adc_mcux_12b1msps_sar.c)
+zephyr_library_sources_ifdef(CONFIG_ADC_MCUX_12B1MSPS_SAR	adc_mcux_12b1msps_sar.c)
```

Comment:
Please avoid combining style changes with functional changes in a single commit; it clutters the diffs and mixes concerns.

Reply:
ok, line reverted